mod/node-api/handlers/beacon/validators.go

```diff
@@ -61,6 +69,20 @@ func (h *Handler[_, ContextT, _, _]) GetStateValidators(
 	}, nil
 }
 
+// CustomValidator to hold the validator data for BeaconAPIs.
+//
+//nolint:lll
+type CustomValidator struct {
```

Comment:
For the validator, similar to `Fork` we created a `ForkResponse` in types/response.go, let's do the same thing. Create a `ValidatorResponse` struct that takes in a `Validator` type (that we define in types/types.go. And just write a `MarshalJSON` function on the `ValidatorResponse` type, which uses the `Validator` object that it holds.

Reply:
Sure, done!